o2a/mappers/spark_mapper.py

```diff
@@ -170,39 +141,21 @@ def _get_tasks(self):
                 main_jar=self.java_jar,
                 main_class=self.java_class,
                 arguments=self.application_args,
-                archives=self.hdfs_archives,
-                files=self.hdfs_files,
+                hdfs_archives=self.hdfs_archives,
```

Comment:
I think you should change the name of this variable to match its purpose in the template. 
```bash
sed -i "/hdfs_archive/archive/" o2a/mappers/spark_mapper.py
```